### PR TITLE
Cleanup python files #1

### PR DESCRIPTION
**Pull request by persianpros**

### CODE REVIEW

File: skin.py [hunk 1]
```diff
@@ -38,7 +38,7 @@ def __init__(self, message):
 		self.msg = message
 
 	def __str__(self):
-		return "{%s}: %s. Please contact the skin's author!" % (config.skin.primary_skin.value, self.msg)
+		return "{%s}: %s. Please contact the skin's author!" % config.skin.primary_skin.value, self.msg
```
Comment (athoik) [inline skin.py]:
```
>>> "{%s}: %s. Please contact the skin's author!" % 1, 2
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not enough arguments for format string
>>> "{%s}: %s. Please contact the skin's author!" % (1, 2)
"{1}: 2. Please contact the skin's author!"
```

**Comment by littlesat** [comment]
This is not good...

-		return "{%s}: %s. Please contact the skin's author!" % (config.skin.primary_skin.value, self.msg)
 +		return "{%s}: %s. Please contact the skin's author!" % config.skin.primary_skin.value, self.msg

**Comment by persianpros** [comment]
Updated, Now you can merge this :)

**Comment by littlesat** [comment]
Did you test it?..... Because the mistake you previous made I need to test it myself before I can merge it....

**Pull request closed without merging**